app/helpers/application_helper/button/vm_publish.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationHelper::Button::VmPublish < ApplicationHelper::Button::Basic
   needs :@record
 
   def visible?
-    @record.is_available?(:publish) && !@is_redhat
+    @record.supports?(:publish)
```

Comment:
@durandom Yep, no need to keep `VmPublish`. @cynepco3hahue please remove this button, and use [GenericFeatureButton](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button.rb), instead :wink:


This is an example of how the usage looks like in a toolbar:
```diff
- :klass => ApplicationHelper::Button::VmPublish
+ :klass => ApplicationHelper::Button::GenericFeatureButton,
+ :options => {:feature => :publish},
```